### PR TITLE
Fabo/1887 track by default

### LINKED ISSUE #1887: switch from opt-in to opt-out for analytics
> Because the user isn't able to opt in. 

i think we need to default to _some amount_ of analytics. i am very open to letting users opt out. re: cookies - we should investigate the requirements here more closely.

_Originally posted by @jbibla in https://github.com/cosmos/voyager/issues/1718#issuecomment-456448624_

### PR DESCRIPTION
**Pull request by faboweb**
Closes #1887

This anonymizes data and enables tracking by default

**Description:**

Thank you! 🚀

---

For contributor:

- [ ] Added entries in `CHANGELOG.md` with issue # and GitHub username
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

File: app/src/renderer/vuex/modules/session.js [hunk 1]
```diff
@@ -208,19 +204,13 @@ export default () => {
           dsn: state.externals.config.sentry_dsn,
           release: state.externals.config.version
         })
-        state.externals.enableGoogleAnalytics(
-          state.externals.config.google_analytics_uid
-        )
-        console.log(`Analytics and error reporting have been enabled`)
+        console.log(`Error collection have been enabled`)
```
Comment (jbibla) [inline app/src/renderer/vuex/modules/session.js]:
```suggestion
        console.log(`Error collection has been enabled`)
```

File: app/src/renderer/vuex/modules/session.js [hunk 2]
```diff
@@ -208,19 +204,13 @@ export default () => {
...
         state.externals.track(`pageview`, {
           dl: window.location.pathname
         })
       } else {
-        console.log(`Analytics disabled in browser`)
+        console.log(`Error collection disabled in browser`)
```
Comment (jbibla) [inline app/src/renderer/vuex/modules/session.js]:
```suggestion
        console.log(`Error collection has been disabled`)
```

**Comment by jbibla** [comment]
what do you mean `enables tracking by default`?

**Comment by faboweb** [comment]
This will track any user that enters the page by default. But will anonymize the data.

Based on: https://www.blastam.com/blog/gdpr-need-consent-for-google-analytics-tracking
And the suggestions by users. Needs to be also checked by legal.

**Pull request merged**